Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -140,15 +148,6 @@ static float toNSURLSessionTaskPriority(WebCore::ResourceLoadPriority priority)
     return statelessCookieStorage.get().get();
 }
 
-#if HAVE(CFNETWORK_CNAME_AND_COOKIE_TRANSFORM_SPI)
-// FIXME: Remove these selector checks when macOS Big Sur has shipped.
-// https://bugs.webkit.org/show_bug.cgi?id=215280
```

Comment:
Should this bug be resolved now?

Reply:
Yep — I'll resolve it as a dupe once this lands.